### DISCUSSION TITLE
DJ Broadcast to Multiple Stations on One Server

### DISCUSSION THREAD

**Discussion by FendleTruck** [AzuraCast/AzuraCast#4742] [Q&A]
Is there any way to have a DJ (using the web dj, or even an external application) to broadcast live on multiple stations?

I have multiple stations running on my server, and I often want to have the same live content sent to two of them.

For example:
Station 1 - Variety Station
Station 2 - Older Content
Station 3 - Only talk Content
Every now and then, I have live talk content that I want to send to the Station 1 and 3.

Is this possible, or do I just air a advertisement to switch stations for the live content?

**Comment by ivellios1988** [comment]
What application your DJs use to connect with your server? Software such as RadioBOSS or SAM Broadcaster usually offers the possibility to broadcast to multiple stations simultaneously.

**Reply by FendleTruck** [reply]
I usually use the webDJ or Butt.

Was just wondering if there was something I could do server side for this, but if it's a DJ-side thing I can look into those suggestions.

**Comment by SpeedyCharly** [comment]
Butt seems to be a 'bare-bone' streamer with scant options, somewhat like Winamp.  webDJ doesn't allow multiple encoders either, you will have to look for a better streaming app.  Have you considered Lunar Caster or Mixxx which are freeware?  Not 100% sure about Mixxx, but I know for a fact that Lunar Caster supports multiple encoders.
